_posts/en/newsletters/2024-01-24-newsletter.md

```diff
@@ -0,0 +1,233 @@
+    relay.  Gregory Sanders [proposes][sanders transition] an
+    intermediate stage with temporary special treatment of transactions
+    that look like current anchors-style LN commitment transactions,
+    allowing Bitcoin Core to deploy cluster mempool and v3.  When
+    they've been widely deployed and all LN implementations have
+    upgraded to using them, the temporary special rules can be dropped.
+
+  - *Request for max child size:* the draft proposal for v3 relay sets
+    the size of an unconfirmed transaction's child to 1,000 vbytes.  The
+    larger this size, the more an honest user will need to pay to
+    overcome a [transaction pin][topic transaction pinning] (see
+    [Newsletter #283][news283 v3pin]).  The smaller the size, the fewer
+    inputs an honest user can use to contribute fees.
+
+- **New documentation repository:** Anthony Towns [posted][towns binana]
+  to the Bitcoin-Dev mailing list to announce a new repository for
+  protocol specifications, _Bitcoin Inquisition Numbers And Names
+  Authority_ ([BINANA][binana repo]).  Four specifications are available
+  in the repository at the time of writing:
+
+    - [BIN24-1][] `OP_CAT` by Ethan Heilman and Armin Sabouri.  See
+      the description of their soft fork proposal in [Newsletter
+      #274][news274 cat].
+
+    - [BIN24-2][] Heretical Deployments by Anthony Towns, describing the
+      use of [Bitcoin Inquisition][bitcoin inquisition repo] for
+      proposed soft forks and other changes on the default signet.  See
+      the extended description in [Newsletter #232][news232 inqui].
+
+    - [BIN24-3][] `OP_CHECKSIGFROMSTACK` by Brandon Black, specifying
+      this [long-proposed idea][topic OP_CHECKSIGFROMSTACK].  See [last
+      week's newsletter][news285 lnhance] for Black's proposal to
+      include this opcode as part of the LNHANCE soft fork bundle.
+
+    - [BIN24-4][] `OP_INTERNALKEY` by Brandon Black, specifying an
+      opcode for retrieving the taproot internal key from the script
+      interpreter.  This was also described in last week's newsletter as
+      part of the LNHANCE soft fork bundle.
+
+  Bitcoin Optech has added the BINANA repository to the list of
+  documentation sources we monitor for updates, which also includes
+  BIPs, BOLTs, and BLIPs.  Future updates may be found in the section
+  for _notable code and documentation changes_.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Envoy 1.5 released:**
+  [Envoy 1.5][] adds support for [taproot][topic taproot] sending and receiving and
+  changes the way [uneconomical outputs][topic uneconomical outputs] are
+  handled in addition to bugfixes and [other updates][envoy blog].
+
+- **Liana v4.0 released:**
+  [Liana v4.0][] was [released][liana blog] and includes support for [RBF fee
+  bumping][topic rbf], transaction cancelling using RBF, automatic [coin
```

Comment:
```suggestion
  bumping][topic rbf], transaction canceling using RBF, automatic [coin
```
(American English spelling; "cancelling" is British English, so I'm not sure)
So far in the newsletters, we have 4 occurrences of "canceling" and zero of "cancelling".